src/queries/match-query.js

```diff
@@ -1,25 +1,26 @@
+import _ from 'lodash'
+
 /**
  * Construct a Match query.
  *
- * @param  {String}  field    Field name to query over.
- * @param  {String}  term     Query value.
- * @param  {Boolean} isPhrase Should the term be treated as a phrase or not.
- * @return {Object}           Match query.
+ * @param  {String} field Field name to query over.
+ * @param  {String} term  Query value.
+ * @param  {Object} opts  See https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html for available options
+ * @return {Object}       Match query.
  */
-export default function matchQuery(field, term, isPhrase = false) {
-  if (isPhrase) {
+export default function matchQuery(field, term, opts = {}) {
```

Comment:
@danpaz This can just be consolidated into what's in the else statement but I left the case when options are not provided since it broke a number of tests. I can make this change if you like.